RecoPixelVertexing/PixelTriplets/python/quadrupletseedmerging_cff.py

```diff
@@ -48,3 +60,4 @@
 )
 eras.trackingPhase1.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
 eras.trackingPhase1PU70.toModify(PixelSeedMergerQuadruplets, **_forPhase1)
+eras.trackingPhase2PU140.toModify(PixelSeedMergerQuadruplets, layerList = _layerListForPhase2)
```

Comment:
In the end you should need `**_forPhase1` too.

Or maybe we should just change the default value of `HitProducer`, as this configuration is first used in phase1 anyway. But even in that case the cleanup is probably better done in a separate PR.